magento2/magento2/scripts/init-and-run.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash -x
+
+# Turn on monitor mode, required by job control
+set -m
+
+if [ ! -f /var/www/html/magento2/.installed ]; then
+  cd /var/www/html/magento2/ || exit
+  composer config --global repos.packagist composer https://packagist.org
+  composer require nosto/php-sdk:@stable
```

Comment:
Why is the SDK set up here?

Reply:
Magento needs to have the SDK before requiring the main extension